### DISCUSSION TITLE
Linux version randomly and repeatedly asks to unlock login keyring?

### DISCUSSION THREAD

**Discussion by osa1** [zed-industries/zed#15960] [Feature Requests]
The Linux version, on startup and then later during use, keep asking to unlock the login keyring.

The problem is I'm not using anything that would require Zed to unlock anything. I'm not logging in to any AI services or anything else. Copilot is disabled: https://github.com/osa1/rcbackup/blob/844e25f7acfc2d95c602ade16c80e786aa5a2507/zed/settings.json#L14-L17

Closing the dialogue with ESC doesn't break anything, it works fine until it asks for it again in a few minutes.

Any thoughts on what may be causing this and how to disable it?  Version 0.147.2.

**Comment by osa1** [comment]
I'm still having this problem and it seems to be getting worse.

For example, I just opened Zed in https://github.com/servo/servo.git and globally searched for `unsafe`, it asked for the password for about 10 times.

**Comment by thekarananand** [comment]
![screenshot-2024-09-14-18-37-16](https://github.com/user-attachments/assets/64b83584-d9bb-4e7e-97f0-076612e947a1)

anyways to fix it

**Reply by prabhat-suresh** [reply]
Yeah, facing the same issue

**Comment by P1n3appl3** [comment]
I'm pretty sure this is the same issue being discussed here: #17460

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
